image/system_a/data/lang/English.json

```diff
@@ -126,9 +126,9 @@
     "app_bell_settings_alarm_settings_title": "Alarm settings",
     "app_bell_settings_alarm_settings_tone": "Main alarm tone",
     "app_bell_settings_alarm_settings_volume": "Main alarm volume",
-    "app_bell_settings_bedtime_settings_tone": "Bedtime tone",
-    "app_bell_settings_bedtime_settings_volume": "<text>Bedtime tone<br />volume</text>",
-    "app_bell_settings_bedtime_tone": "Bedtime tone",
+    "app_bell_settings_bedtime_settings_tone": "Bedtime chime",
+    "app_bell_settings_bedtime_settings_volume": "<text>Bedtime volume</text>",
```

Comment:
Is it this `<text>` tag necessary here?

Reply:
You're right, it's not needed after that `<br/>` was removed, fixed :wink: